packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -545,6 +566,24 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         return result;
     }
 
+    public changeIntervalById(id, start: number, end: number, props?: MergeTree.PropertySet): TInterval | undefined {
+        const interval: TInterval = this.getIntervalById(id);
+        if (interval) {
+            const newInterval = this.changeInterval(interval, start, end, props);
+            return newInterval;
+        }
+    }
+
+    public changeInterval(interval: TInterval, start: number, end: number, props?: MergeTree.PropertySet) {
+        this.removeExistingInterval(interval);
+        const newInterval: TInterval = interval.modify(this.label, start, end);
+        if (props) {
+            newInterval.properties = MergeTree.addProperties(newInterval.properties, props);
```

Comment:
this doesn't look right, these properties also need to be managed for eventual consistency, see SegmentPropertiesManager